### ISSUE TITLE
[terra-form-select] non-selected option does not have high enough contrast

### ISSUE THREAD

**Issue by dv297** [cerner/terra-core#1917]
# Bug Report

## Description
When running aXe on [terra-form-select's doc page](https://engineering.cerner.com/terra-ui/#/raw/components/terra-form-select/form-select/select), this "serious"-level problem occurs.

Ensures the contrast between foreground and background colors meets WCAG 2 AA contrast ratio thresholds

## Steps to Reproduce
1. Ensure select has non-selected option selected
2. Run aXe test

## Additional Context / Screenshots
Example of terra-form-select with nothing selected
![terra-form-select](https://user-images.githubusercontent.com/5365325/45902858-784b6900-bdad-11e8-95ef-d4102b6eb3c6.PNG)

## Expected Behavior
aXe test should pass

## Possible Solution
Increase contrast

## Environment

* Component Name and Version: terra-form-select@4.17.1
* Browser Name and Version: Chrome Desktop Version 69.0.3497.100 (Official Build) (64-bit)
* Operating System and version (desktop or mobile): Windows 10 Enterprise Desktop

**Comment by StephenEsser** [comment]
My understanding from previous conversations with UX was that certain non-interactive or disabled content was not required to pass WCAG 2 AA contrast ratio, but was required to pass another contrast ratio standard. 

That said, I don't think there is a way to circumvent this aXe failure so if there is a way to bring the placeholder text into the right contrast threshold to pass this test I think we should, assuming UX agrees with the change and can provide a new set of colors. 

cc: @neilpfeiffer

**Comment by bjankord** [comment]
@scottwilmarth Do you have any thoughts on this?

**Comment by bjankord** [comment]
`#757575` seems like a good choice. It is one of the lightest greys that when displayed on a white background, pass WCAG AA Normal and Large text contrast ratio guidelines. I understand the desire not to have placeholder text look like the value text, but at the same time, the placeholder text should still be easy to see/read.
